jcommune-performance-tests/README.md

```diff
@@ -0,0 +1,24 @@
+#Gatling performance tests ##
+
+##HOW TO ##
+
+1. Run your application server used for testing.
+2. Configure pom.xml
+    ```
+    <configuration>
+        <runMultipleSimulations>false</runMultipleSimulations> (1)
+        <simulationClass>org.jtalks.jcommune.performance.tests.OpenTopicPage</simulationClass> (2)
+        <jvmArgs>
```

Comment:
Instead of changing pom.xml itself it's possible to create variables (like `performance.url`) in and pass them to Maven: `<jvmArg>-Durl=${performance.url}</jvm>`. Then to run the tests you'd do: `mvn test -Dperformance.url=...`.

By default I think it's safe to use http://performance.jtalks.org